frontend/src/lib/components/LemonTable/LemonTable.tsx

```diff
@@ -304,8 +305,8 @@ export function LemonTable<T extends Record<string, any>>({
                                             </th>
                                         ))
                                     )}
+                                    <LemonTableLoader loading={loading} />
                                 </tr>
-                                <tr className="LemonTable__loader" />
```

Comment:
I wanted the same loader as it looks nice for the recordings table but using CSS like this would be hacky hence bringing it out to its own component so it can be used in the LemonTable as well as the Recordings List.

Reply:
Ah that makes sense! I didn't see it was being used in two different places :)